README.md

```diff
@@ -1,3 +1,90 @@
 # repo-template
 
 **This is not an official Google product.**
+
+This template is for creating an AOD (access on demand) repository to get access
+to GCP resources on demand.
+
+## Make an AOD Request
+
+**Make sure you have completed
+[prerequisite steps](https://github.com/abcxyz/aod-template/blob/main/README.md#prerequisites)
+to set up this repo properly.**
+
+1.  Open a PR
+
+    -   A title and description to provide enough justification. E.g. "AOD
+        request: debug customer issue X".
+    -   To request IAM permissions on org/folder/project level, add an
+        `iam.yaml` file in the repo root.
+    -   See an example of this file, see example in
+        [example-iam.yaml](https://github.com/abcxyz/aod-template/example-iam.yaml).
+    -   Optionally use predefined duration labels on the PR to specify the IAM
+        permission expiration. Otherwise a 2h default duration will be used.
+    -   (Later) To request on-demand `gcloud` commands, add an `gcloud.yaml`
```

Comment:
Do we have an example of `gcloud.yaml`?

Reply:
Not really, it is to be implemented. The design doc has a rough example, but since this feature is not available yet, plan to add that when it id done.